GDJS/Runtime/howler-sound-manager/howler-sound-manager.ts

```diff
@@ -132,37 +154,35 @@ namespace gdjs {
      * Get the sound playback rate.
      */
     getRate(): float {
-      if (this._id === null) return 0;
-      return this._howl.rate(this._id);
+      return this._rate;
     }
 
     /**
      * Set the playback rate.
-     * @returns The current instance for chaining.
      */
-    setRate(rate: float): this {
+    setRate(rate: float): void {
+      this._rate = rate;
+      // If the sound has already started playing, then change the value directly.
       if (this._id !== null) {
         rate = gdjs.HowlerSoundManager.clampRate(rate);
         this._howl.rate(rate, this._id);
       }
```

Comment:
It's important to keep the return this because this allow to chain calls ("fluent" interface) - this would be a breaking change to change this.

Reply:
As I removed the calls to theses functions from the file itself, I though it wasn't useful anymore. I'll add it back